flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorTest.java

```diff
@@ -281,20 +283,7 @@ public void testShouldShutDownTaskManagerServicesInPostStop() throws Exception {
 			.setTaskStateManager(localStateStoresManager)
 			.build();
 
-		final long heartbeatInterval = 1000L;
-		final long heartbeatTimeout = 1000L;
-		final HeartbeatServices heartbeatServices = new HeartbeatServices(heartbeatInterval, heartbeatTimeout);
-
-		final TaskExecutor taskManager = new TaskExecutor(
-			rpc,
-			taskManagerConfiguration,
-			haServices,
-			taskManagerServices,
-			heartbeatServices,
-			UnregisteredMetricGroups.createUnregisteredTaskManagerMetricGroup(),
-			null,
-			dummyBlobCacheService,
-			testingFatalErrorHandler);
+		final TaskExecutor taskManager = createTaskExecutor(taskManagerServices);
```

Comment:
NIce refactoring for TaskExecutor creation, could you move it into a separate commit?